R/expect_not_linter.R

```diff
@@ -22,10 +22,10 @@
 #' @evalRd rd_tags("expect_not_linter")
 #' @seealso [linters] for a complete list of linters available in lintr.
 #' @export
-expect_not_linter <- make_linter_from_xpath(
+expect_not_linter <- make_linter_from_function_xpath(
```

Comment:
Is it too much overload to just add an argument to `make_linter_from_xpath()` to handle this instead of a new function?

Reply:
It would require casework in the `Linter()` itself, harming performance. Or some `eval(bquote())` trickery.
I'd rather not do either of those.

Reply:
SGTM, thanks